react/README.md

```diff
@@ -11,34 +11,58 @@ Mayflower-react is a React component library under the [enterprise design system
 
 
 ## Using Mayflower-React in Your Project
-1. Install mayflower-react into your project as a dependency.
-`npm i @massds/mayflower-react --save`
-2. Import components into your App.js
-`import { Button } from '@massds/mayflower-react';`
-3. Render components in JSX
-`<Button text="Button" onClick={() => console.log('mayflower button clicked!')} />`
-
-Please note that [@massds/mayflower-react][npm] comes with prebuilt css, svg and other image files. If you are configuring your own Webpack, you might need to add the following loaders to handle these files:
-``npm install --save-dev css-loader svg-inline-loader file-loader``
-
-You can also kickstart your React project with Mayflower-react referring to the [Mayflower React Starter][react-starter] built with [Create React App](https://reactjs.org/docs/create-a-new-react-app.html#create-react-app).
-
-For a guide and information on the components included in mayflower-react and their functionality, visit our [Mayflower React Storybook][react-storybook]. Click on the Info and Knobs tabs for component prop types, details and options.
+1. Install mayflower-react and mayflower-assets into your project dependency:
+```javascript
+npm i @massds/mayflower-react @massds/mayflower-assets --save
```

Comment:
let's also make `@massds/mayflower-assets` a peer dependency in `@massds/mayflower-react`, if we are telling people to install them separately